src/org/traccar/Context.java

```diff
@@ -413,9 +423,17 @@ public static void init(String[] arguments) throws Exception {
 
     }
 
+    private static EventForwarder getConfiguredEventForwarder() {
```

Comment:
I don't think we need a separate method for this.

Reply:
Method length would be 154 lines (max allowed is 150)

Reply:
You need to find a better way to fix it and split original method into meaningful chunks.

Reply:
Done